src/components/Pagination.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+	<div class="pagination-container">
+		<b-button-group>
+			<b-button
+				v-for="page in pages"
+				:id="'tab'+page.index"
+				:key="page.index"
+				class="pagination-button"
+				:style="cssVars"
+				:pressed="selectedIndex == page.index"
+				@click="handleClick(page)"
+			>
+				{{ page.value }}
+			</b-button>
+		</b-button-group>
+	</div>
+</template>
+
+<script>
+
+export default {
+	props: {
+		perPage: {
+			type: Number,
+			default: 5,
+			description: `The maximum number of elements
```

Comment:
It is possible to add a validation on this prop? It should be at least 1.

Reply:
Even with the validation, when an invalid value is passed, it still ends up with infinity tabs. So I added a verification where if a value lower than 1 is passed, it is ignored and the component's behavior is the same as if that value was 1.